app/frontend/javascript/google-tag/index.js

```diff
@@ -67,3 +67,19 @@ export function attachExternalLinkTracker () {
     })
   })
 }
+
+export function attachQuestionXsRoutesTracker () {
+  const showRoutesPathRegex = /^\/forms\/\d+\/pages\/\d+\/routes$/
```

Comment:
just to explain, this is going to match a path that goes `/forms/form_id/pages/page_id/routes`